app/grants/models.py

```diff
@@ -272,6 +272,49 @@ def org_profile(self):
             return profiles.first()
         return None
 
+    @property
+    def history_by_month(self):
+        # gets the history of contributions to this grant month over month so they can be shown o grant details
+        # returns [["", "Subscription Billing",  "New Subscriptions", "One-Time Contributions", "CLR Matching Funds"], ["December 2017", 5534, 2011, 0, 0], ["January 2018", 10396, 0 , 0, 0 ], ... for each monnth in which this grant has contribution history];
+        month_to_contribution_numbers = {}
+        for sub in self.subscriptions.all():
+            for contrib in sub.subscription_contribution.filter(success=True):
+                #add all contributions
+                key = contrib.created_on.strftime("%Y/%m")
+                subkey = 'One-Time' 
+                if int(contrib.subscription.num_tx_approved) > 1:
+                    if contrib.is_first_in_sequence:
+                        subkey = 'New-Recurring'
+                    else:
+                        subkey = 'Recurring-Recurring'
+                if contrib.subscription.contributor_profile.handle in ['vs77bb', 'gitcoinbot', 'notscottmoore', 'owocki']:
```

Comment:
thoughts on moving this to a constant defined above ?
['vs77bb', 'gitcoinbot', 'notscottmoore', 'owocki']`

Reply:
updated